.github/rawWorkflows/gh-ci-parameterized-flow.txt

```diff
@@ -28,3 +28,11 @@
         uses: gradle/gradle-build-action@v2
         with:
           arguments: "$GradleArguments"
+      - name: Upload test logs on failure
+        if: ${{ failure() }}
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ github.job }}.logs
+          path: internal/venice-test-common/build/test-results/*/TEST-*.xml
```

Comment:
I don't like this very much since it adds this path for unit tests jobs too. Why not use `**/build/test-results/*/TEST-*.xml`?

Reply:
Good point.

I wanted to skip log uploads for unit tests as they don't add much value.  I tried to do it with the conditional job name matching but somehow that didn't work.   Hence I was relying on path name matching to not upload logs. 


But I 100% agree with you. I'll update PR with the name  `**/build/test-results/*/TEST-*.xml`.

Reply:
Updated PR with generic path but also added conditional to prevent log uploads for non-Integration tests

Reply:
But why do acrobatics to prevent unit test logs from uploading? Is the action name accurate, that this will only upload on failures? Or does it upload everytime? Either way, I think it would be fine to upload everything without distinction...

Reply:
> But why do acrobatics to prevent unit test logs from uploading?
To avoid hitting the GH storage limit. 

> Is the action name accurate, that this will only upload on failures?

Yeah since we run upload action conditionally: 
```
      - name: Upload test logs on failure
        if: failure() && startsWith(github.job, 'IntegrationTest')
        uses: actions/upload-artifact@v3
```

Reply:
> Either way, I think it would be fine to upload everything without distinction...

I can remove these conditional uploads. We can always re-enable them if we hit GH storage limits